### DISCUSSION TITLE
Tooltip for pie chart

### DISCUSSION THREAD

**Discussion by danh7598** [Abhinandan-Kushwaha/react-native-gifted-charts#861] [Ideas]
Hello, it would be great if you could add support for rendering a tooltip when focusing on each pie slice, similar to how it works in the bar chart.
![image](https://github.com/user-attachments/assets/99bb6056-067a-441e-9d7a-f984e368ea62)

**Comment by Abhinandan-Kushwaha** [comment]
Thanks for requesting this feature @danh7598 
I will add it soon.

**Comment by Abhinandan-Kushwaha** [comment]
Hi @danh7598 
This feature is now available from versions `1.4.44` onwards. 🎉

See the [tooltip related props here](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/blob/master/docs/PieChart/PieChartProps.md#tooltip)

Please use the latest version of the library.
